src/account/discordAccount.mongo.service.ts

```diff
@@ -27,7 +27,7 @@ export class DiscordAccountMongoService {
 
     async findManyAccount(filter): Promise<DiscordAccount[] | null> {
         try {
-            return await this.discordAccountModel.find(filter).exec().catch((e) => {
+            return await this.discordAccountModel.find(filter).lean().exec().catch((e) => {
```

Comment:
Curious, what does .lean do?

Reply:
returns a simple js objects instead of mongoose document. In some cases easier to handle.